src/wincng.h

```diff
@@ -299,21 +302,20 @@ struct _libssh2_wincng_cipher_ctx {
 struct _libssh2_wincng_cipher_type {
     BCRYPT_ALG_HANDLE *phAlg;
     unsigned long dwKeyLength;
-    unsigned long dwUseIV;
+    int useIV;
+    int ctrMode;
```

Comment:
Thanks for your changes. What do you think about using bool instead of int? I know that bool usually maps to int on most plattforms/toolchains, but the compiler could optimize it since only a boolean value is stored.